articles/azure/azs-how-create-service-principals.md

```diff
@@ -0,0 +1,97 @@
+---
+title: How to create a service principal for Azure AD 
+description: Give applications access to Azure Stack resources by creating service principals
+services: azure-stack
+author: David Woffendin
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: How to create a service principal for Azure AD 
```

Comment:
Take the How to out here, so you have:

Create a service principal for Azure AD